apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/page.tsx

```diff
@@ -37,15 +40,28 @@ export default async function ConnectedEvaluationPage({
     projectId: Number(params.projectId),
     uuid: params.commitUuid,
   })
-  const { rows, pagination } = await paginateQuery({
```

Comment:
i've ejected from paginatequery because it's based on the assumption that pagination is the last step of the process whereas i want it to be the first. If this works we will explore a way to DRY a bit everything.